### ISSUE TITLE
Checking for insufficient conditions

### ISSUE THREAD

**Issue by sankarshanmukhopadhyay** [red-hat-storage/openshift-cic#35]
I attempted to input wrong/insufficient data to check if we do get any output. I have pasted the output below. This is off a clone (from the master)

```
./cic.py 
------------------------------------------------------------
   CNS - Inventory File Creator
------------------------------------------------------------
'' 
This tool is very highly version dependent. It is currently,
configured to work with OpenShift 3.9 and CNS 3.9.
The output is NOT A COMPLETE Inventory File.
Created Inventory file options should be copied and pasted into
the larger openshift-ansible inventory file for your deployment.
 ''
1. Storage for Applications + Registry 
2. Storage for Applications + Logging
3. Storage for Applications + Metrics 
4. Storage for Applications + Registry + Logging + Metrics
5. Storage for Applications Only
------------------------------------------------------------
Enter your choice [1-5] : 1
------------------------------------------------------------
 For this configuration 4 nodes are recommended 
 With a minimum of 3 required.
------------------------------------------------------------
How many nodes are available ?:  1
What hosts will be used for application storage (IP/FQDN) ?:  127.0.0.1
What are the raw storage devices for these hosts (/dev/<device>) ?: /dev/sdx
What is the size of each raw storage device (GB) ?: 10
What is the size for the registry persistent volume (GB)?: 10
    
[OSEv3:children]
glusterfs
   
[OSEv3:vars]
# registry
openshift_hosted_registry_storage_kind=glusterfs
openshift_hosted_registry_storage_volume_size=10Gi
openshift_hosted_registry_selector="region=infra"
   
# Container image to use for glusterfs pods
openshift_storage_glusterfs_image=registry.access.redhat.com/rhgs3/rhgs-server-rhel7
openshift_storage_glusterfs_version=v3.9
  
# Container image to use for glusterblock-provisioner pod
openshift_storage_glusterfs_block_image=registry.access.redhat.com/rhgs3/rhgs-gluster-block-prov-rhel7
openshift_storage_glusterfs_block_version=v3.9
  
# Container image to use for heketi pods
openshift_storage_glusterfs_heketi_image=registry.access.redhat.com/rhgs3/rhgs-volmanager-rhel7
openshift_storage_glusterfs_heketi_version=v3.9
    
# CNS storage cluster
openshift_storage_glusterfs_namespace=app-storage
openshift_storage_glusterfs_storageclass=true
openshift_storage_glusterfs_storageclass_default=false
openshift_storage_glusterfs_block_deploy=false
   
[glusterfs]
127.0.0.1 glusterfs_zone=1 glusterfs_devices='["/dev/sdx"]'
```

**Comment by prernasony** [comment]
I submitted PR which fixes a part of this issue .
PR works to check if number of nodes are less than 3

**Comment by netzzer** [comment]
Closed because of PR #36.
